src/gigs-board/pages/community/telegram.jsx

```diff
@@ -135,6 +135,16 @@ const Telegram = (
         minWidth: "100%",
       }}
     ></iframe>
+
+    <a href={"https://t.me/" + communityData.telegram_handle} target="_blank">
```

Comment:
My bad, I've forgotten about link buttons upon button atom implementation. I would be grateful if you create a ticket for enabling support for `<a />` as a root node in `components.atom.button`!

But as I can see, the current implementation doesn't cause issues, so I'd like to address it later. LGTM!